manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -130,6 +119,17 @@ spec:
         severity: warning
   - name: openshift-etcd.rules
     rules:
+    - alert: etcdGRPCRequestsSlow
+      annotations:
+        description: 'etcd cluster "{{ $labels.%s }}": 99th percentile of gRPC requests is {{ $value }}s on etcd instance {{ $labels.instance }} for {{ $labels.grpc_method }} method.'
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdGRPCRequestsSlow.md
+        summary: etcd grpc requests are slow
+      expr: |
+        histogram_quantile(0.99, sum(rate(grpc_server_handling_seconds_bucket{job="etcd", grpc_method!="Defragment", grpc_type="unary"}[10m])) without(grpc_type))
+        > 1
```

Comment:
We collect [a bunch of etcd performance metrics][1] in Telemetry.  Maybe we can process those, and possibly also correlate to other signs of cluster distress, to motivate particular thresholds?  No worries if we want to take a guess now, and circle back later when we have more time for analysis.

[1]: https://github.com/openshift/cluster-monitoring-operator/blame/492c01da6f0d16bdb046d7736da484e256c582df/Documentation/data-collection.md#L608-L629

Reply:
we have this in our backlog already in:
https://issues.redhat.com/browse/ETCD-144

feel free to add anything that's missing from your PoV.

Reply:
as for that threshold, we don't gather the grpc latencies sadly, so there's nothing much we can gather from telemeter unfortunately.